### DISCUSSION TITLE
The project is great, but the license is very bad

### DISCUSSION THREAD

**Discussion by trusktr** [yandeu/five-server#116] [General]
It says no one can use the software, which makes it closed source (even if the code is *visible* on GitHub).

> ```
> 2. RESTRICTIONS
> You are NOT permitted to:
>   a)  Edit, alter, modify, adapt, translate or otherwise change the whole or any 
>       part of the Software without the express permission of the Licensor. 
>   b)  Decompile, disassemble or reverse engineer the Software or attempt to do 
>       any such things.
>   c)  Reproduce, copy, distribute, resell or otherwise use the whole or any part 
>       of the Software for any commercial or non-commercial purpose.
>   d)  Disable, modify or hide notifications sent by the Software.
> ```

People cannot do anything with the source, and they can't even use it for anything at all!

Based on the license, this code is 100% closed source software, and if someone uses it, you can sue them.

**Comment by yandeu** [comment]
Yes, it is a bit strict. What do you want to use it for?

**Reply by trusktr** [reply]
I was using live-server to serve local static websites during development, but it had issues in Windows while five-server works in Windows.

**Reply by yandeu** [reply]
You can use it for that without any issues. I have published it on npm and vscode. Of course you can use it for your use case.
